magento/magento2
# Duplicating Configurable Product With Related Products Not Working Magento 2.1.7 #69738

### Description


### Fixed Issues (if relevant)

1. magento/magento2#9851: Duplicating Configurable Product With Related Products Not Working Magento 2.1.7 #9851
2. Skip deletion of custom option If product is from duplication

### Manual testing scenarios

1. Assign some related product to configurable product and clicked on "Save and Duplicate" to confirm and after duplicating product I can see related product in new duplicated product
2. Also, check on different product types and Its working fine

### Contribution checklist
